SPECS-EXTENDED/libreswan/libreswan.spec

```diff
@@ -0,0 +1,492 @@
+    USE_LIBCAP_NG=true \\\
+    USE_LIBCURL=true \\\
+    USE_LINUX_AUDIT=true \\\
+    USE_NM=true \\\
+    USE_NSS_IPSEC_PROFILE=true \\\
+    USE_SECCOMP=true \\\
+    USE_AUTHPAM=true \\\
+%{nil}
+
+Summary:        Internet Key Exchange (IKEv1 and IKEv2) implementation for IPsec
+Name:           libreswan
+Version:        4.7
+Release:        2%{?dist}
+License:        GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Libraries
+Url:            https://libreswan.org/
+Source0:        https://github.com/libreswan/libreswan/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+%if %{with_check}
+Source3:        https://download.libreswan.org/cavs/ikev1_dsa.fax.bz2
+Source4:        https://download.libreswan.org/cavs/ikev1_psk.fax.bz2
+Source5:        https://download.libreswan.org/cavs/ikev2.fax.bz2
+%endif
+
+BuildRequires: audit-libs-devel
+BuildRequires: bison
+BuildRequires: curl-devel
+BuildRequires: flex
+BuildRequires: gcc
+BuildRequires: gnupg2
+BuildRequires: hostname
+BuildRequires: ldns-devel
+BuildRequires: libcap-ng-devel
+BuildRequires: libevent-devel
+BuildRequires: libseccomp-devel
+BuildRequires: libselinux-devel
+BuildRequires: make
+BuildRequires: nspr-devel
+BuildRequires: nss-devel >= %{nss_version}
+BuildRequires: nss-tools >= %{nss_version}
+BuildRequires: openldap-devel
+BuildRequires: mariner-release
+BuildRequires: pam-devel
+BuildRequires: pkgconfig
+BuildRequires: systemd-devel
+BuildRequires: unbound-devel >= %{unbound_version}
+BuildRequires: xmlto
+Requires: iproute >= 2.6.8
+Requires: nss >= %{nss_version}
+Requires: nss-softokn
+Requires: nss-tools
+Requires: unbound-libs >= %{unbound_version}
+Requires: logrotate
+# for pidof
+Requires: procps-ng
+
+Requires(post): bash
+Requires(post): coreutils
+Requires(post): systemd
+Requires(preun): systemd
+Requires(postun): systemd
+
+%description
+Libreswan is a free implementation of IPsec & IKE for Linux.  IPsec is
+the Internet Protocol Security and uses strong cryptography to provide
+both authentication and encryption services.  These services allow you
+to build secure tunnels through untrusted networks.  Everything passing
+through the untrusted net is encrypted by the ipsec gateway machine and
+decrypted by the gateway at the other end of the tunnel.  The resulting
+tunnel is a virtual private network or VPN.
+
+This package contains the daemons and userland tools for setting up
+Libreswan.
+
+Libreswan also supports IKEv2 (RFC7296) and Secure Labeling
+
+Libreswan is based on Openswan-2.6.38 which in turn is based on FreeS/WAN-2.04
+
+%prep
+%setup -q -n libreswan-%{version}
+# enable crypto-policies support
+sed -i "s:#[ ]*include \(.*\)\(/crypto-policies/back-ends/libreswan.config\)$:include \1\2:" configs/ipsec.conf.in
+sed -i "s/SUBDIRS += ipcheck/#SUBDIRS += ipchec/" testing/programs/Makefile
+%autopatch -p1
+
+%build
+make %{?_smp_mflags} \
+%if 0%{with_development}
```

Comment:
Why is there an extra '0' in this if statement? Is it a typo?

Reply:
Not a typo. It will evaluate to 0 if the macro with_development is undefined :). (Although typically looks like %if 0%**?**{with_development} )